src/repeater/repeater.js

```diff
@@ -238,7 +253,9 @@ function createGeneratorRepeater(generator, func) {
   return create({
     name: generator.name,
     interval: 1000 * generator.intervalSecs, // convert to millis
-    func: () => func(generator).catch(onFailure),
+    func: () => func(generator).catch((err) => {
+      onFailure(err, generator);
```

Comment:
`.catch(onFailure)` is shorthand for `.catch((err) => { return onFailure(err); })` so your change here is essentially removing that `return` call. Is that intentional?

Reply:
No, I do not intend to remove the return. Let me make that change.